config/chips/F401xD_xE.chip

```diff
@@ -9,6 +9,6 @@ flash_pagesize 0x4000        // 16 KB
 sram_size 0x18000            // 96 KB
 bootrom_base 0x1fff0000
 bootrom_size 0x7800          // 30 KB
-option_base 0x0
-option_size 0x0
+option_base 0x40023C14
+option_size 0x10
```

Comment:
I must be missing something obvious, sorry for the noise. At 0x40023c14 there is the `FLASH_OPTCR` register, which is used for option byte access. But it seems this just a single register, so shouldn’t we have four bytes there?

Reply:
I agree on the size. The `FLASH_OPTCR` register has 32 Bits where 24 are user programmable. So the overall size of 4 Byte is the correct one.